pkgs/tools/text/crowdin-cli/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchurl
+, gawk
+, git
+, gnugrep
+, jre
+, unzip
+}:
+
+let
+  pname = "crowdin-cli";
+  version = "3.6.4";
+in stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchurl {
+    url = "https://github.com/crowdin/${pname}/releases/download/${version}/${pname}.zip";
+    sha256 = "123mv0s1jppidmwsvr8a6f8429xmpskxmnv4p8jpnfa9zrw86aaw";
+  };
+
+  nativeBuildInputs = [ unzip ];
+
+  propagatedBuildInputs = [
+      gawk
+    gnugrep
+  git
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D crowdin-cli.jar $out/lib/crowdin-cli.jar
```

Comment:
Not sure if we should put this into a subdirectory under lib.

Reply:
I looked at the `umlet` package and the jar is directly under `lib/` but I don't know if that is a good example.